source/documentation/deploying_services/elasticsearch.md

```diff
@@ -218,11 +218,11 @@ Aiven provide a Kibana user interface through which you can explore the data in
     cf ssh -L 4430:HOSTNAME_FROM_STEP_1:443 NAME_OF_AN_APP_YOU_CAN_SSH_TO
     ```
 
-    Elasticsearch instances are only accessible from the PaaS. This sets up a tunnel so that you can get access, similar to how we use [Conduit](https://github.com/alphagov/paas-cf-conduit) elsewhere.
+    Elasticsearch instances are only accessible from GOV.UK PaaS. This sets up a tunnel so that you can get access, similar to how we use [Conduit](https://github.com/alphagov/paas-cf-conduit) elsewhere.
```

Comment:
What does 'this' refer to here?

Reply:
It's referring to the above `cf ssh` command

Reply:
It's probably worth stating it to avoid any confusion - 'using the cf ssh command sets up a tunnel...'?

Reply:
That sounds very sensible